components/forms/TextPage/TextPage.tsx

```diff
@@ -44,17 +44,17 @@ const PageContent = ({ pageText, urlQuery }: PageContextProps) => {
 
 export const TextPage = (props: TextPageProps): React.ReactElement => {
   const { i18n } = useTranslation("confirmation");
-  const { formConfig, htmlEmail } = props;
+  const { formRecord, htmlEmail } = props;
   const language = i18n.language as string;
```

Comment:
In the destructure I provided as an example we're not modifying the object or the type we're just unpacking the nested  properties we need for the function.  Basically just declaring the const `endPage` as part of the destructure so we can call it directly instead of always using `formRecord.formConfig.form.endPage` in the functions scope.